project/Dependencies.scala

```diff
@@ -235,14 +235,15 @@ object CatalogueDependencies {
       WellcomeDependencies.typesafeLibrary
 
   val idminterDependencies: Seq[ModuleID] =
-    ExternalDependencies.mockitoDependencies ++
-      ExternalDependencies.mySqlDependencies ++
+    ExternalDependencies.mySqlDependencies ++
       ExternalDependencies.circeOpticsDependencies
 
   val matcherDependencies: Seq[ModuleID] =
     ExternalDependencies.mockitoDependencies ++
       ExternalDependencies.scalaGraphDependencies
 
+  val mergerDependencies: Seq[ModuleID] = Nil
```

Comment:
Why specify dependencies as nil? This seems odd to me – I would rather omit it entirely in `build.sbt`, so it's obvious it has no extra dependencies.